www/js/i18n.js

```diff
@@ -37,24 +34,29 @@ var i18n = {
 
     getLanguageInUse: function() {
         var lang = undefined;
-        jQuery.ajax({
-            url: '/api/config',
-            data: {},
-            type: 'GET',
-            dataType: 'json',
-            success: function(data) {
-                lang = data['www.lang'];
-            },
-            async: false  // XXX
-        });
+        try {
+            jQuery.ajax({
+                url: '/api/config',
+                data: {},
+                type: 'GET',
+                dataType: 'json',
+                success: function(data) {
+                    lang = data['www.lang'];
+                },
+                async: true
+            });
+        } catch (err) {
+            console.error(err);
```

Comment:
AFAIK console.something() is not supported by IE. Could you please check whether this has changed or not? I don't want people with IE to be unable to use Neubot's web interface.

Also, what's the whole point of catching and suppressing an exception here? I think you should read the documentation of jQuery.ajax() to understand what is the proper way to route errors.